client/src/pages/ScheduleCalendar/style.js

```diff
@@ -0,0 +1,39 @@
+const ScheduleCalendarStyle = () => ({
+    scheduleCalendarContainer : {
+        display: "flex",
+        margin  : "auto",
+        width : "1000px",
+        height : "480px",
+        "-webkit-box-shadow" : "1px 1px 10px 3px rgba(196,196,196,1)",
+        font : "Arial, Helvetica, sans-serif",
+        "font-weight" : "bold",     
+        "text-align" : "center"
+    },
+    title : {
+        display : "flex",
+        "font-size" : "20px",
```

Comment:
i won't comment on ever line but you get the pattern :)

Reply:
Got this!